index.bs

```diff
@@ -3280,7 +3286,7 @@ and at least one of the following three conditions is true:
                     <span>callback function</span>
             </div></th>
                 <th><div>
-                    <span>dictionary</span>
+                    <span>[open] dictionary</span>
```

Comment:
FWIW, #183 will simplify this too: once it lands, I'll put OpenDictionary/record in the dictionary-like category.